65black/so-lite-react
# #162832552 Get Profile Functionality

#### What does this PR do?
This PR adds the ability for a logged in user to retrieve their profile by navigating to a URL.

#### Description of Task to be completed?
- give a logged in user to retrieve their profile
- write tests for new feature
- add spinner and questionResults components
- write tests for new components

#### How should this be manually tested?
- Clone the repo, run `npm install` and run `npm start`
- Login to your account #3 and navigate to `localhost:7000/profile`
- You should see your profile details and stats

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#162832552

#### Screenshots (if appropriate)
<img width="1440" alt="screenshot 2018-12-27 at 4 55 35 pm" src="https://user-images.githubusercontent.com/36507972/50486229-47588100-09f9-11e9-9087-77e8edb21fbe.png">

#### Questions:
N/A
